airgun/browser.py

```diff
@@ -494,40 +468,34 @@ def _create_container(self):
         if self.container:
             return
         image_version = selenium.__version__
-        if not self._client.images(
-                name=self._get_image_name(image_version)):
-            LOGGER.warning('Could not find docker-image for your'
-                           'selium-version "%s"; trying with "latest"',
-                           self._get_image_name(image_version))
+        if not self._client.images(name=self._get_image_name(image_version)):
+            LOGGER.warning(
+                'Could not find docker-image for your' 'selium-version "%s"; trying with "latest"',
+                self._get_image_name(image_version),
+            )
             image_version = 'latest'
-            if not self._client.images(
-                    name=self._get_image_name(image_version)):
+            if not self._client.images(name=self._get_image_name(image_version)):
                 raise DockerBrowserError(
-                    'Could not find docker-image "%s"; please pull it' %
-                    self._get_image_name(image_version)
+                    'Could not find docker-image "%s"; please pull it'
+                    % self._get_image_name(image_version)
```

Comment:
f-string or format please

Reply:
Found and fixed all remaining uses of % for formatting.